### DISCUSSION TITLE
JsonParseError

### DISCUSSION THREAD

**Discussion by FlorentAdell** [witsa/synapps#652] [Q&A]
Bonjour,

J'ai un soucis quand je veux publier ma synapps
Il me fait une erreur vers 44% "JsonParseError"

![image](https://github.com/witsa/synapps/assets/46195964/d3718e56-0130-4b84-8bbf-0ebb5a3b905f)

**Comment by WITNicolas** [comment]
Salut,
A tous les coups c'est une clé en doublon...
Si tu te mets en F12 (mode Debugg de STUDIO) je pense que tu dois voir où est ce que ça bloque.

**Reply by FlorentAdell** [reply]
![image](https://github.com/witsa/synapps/assets/46195964/e50a09c5-1f69-461b-8d6f-f515cff4ce09)

**Reply by WITNicolas** [reply]
Regarde plutot dans NETWORK au lieu de CONSOLE (c'est juste aprés SOURCE)
Tu verras les Put / Create

**Reply by manufitoussiwit** [reply]
Salut, 
En général, c'est un pb avec une image qui ne passe pas dans le REDY au moment de la publication uniquement. 
Mais ça tombe bien, on vient de sortir la version `1.2.0 de Studio et la version `14.5.0` du REDY qui corrigent ce problème en plus d'accélérer fortement la publication.

**Reply by FlorentAdell** [reply]
ok parfait !

**Comment by manufitoussiwit** [comment]
Lié à l'issue #589
